### PR TITLE
Use serilog, improve log output for bulk publish webjob

### PR DESCRIPTION
**Pull request by timabell**
This is an attempt to invoke the dark magic of application insights in azure

Haven't actually managed to see the output locally yet, but it might be fine...

### CODE REVIEW

File: src/ManageCourses.CourseExporterUtil/Publisher.cs [hunk 1]
```diff
@@ -116,7 +122,7 @@ private static List<SearchCourse> ReadAllCourseData(IManageCoursesDbContext cont
 
                 if (!mappedCourse.CourseSubjects.Any())
                 {
-                    Console.WriteLine(
+                    _logger.Information(
```
Comment (defong) [inline src/ManageCourses.CourseExporterUtil/Publisher.cs]: Should this be info?

**Comment by timabell** [comment]
Rebased and de-odourized

**Comment by timabell** [comment]
[Alerting when bulk publishing has failed](https://trello.com/c/c97yYXEq/366-alerting-when-bulk-publishing-has-failed)

**Comment by timabell** [comment]
[better logging for "bulk publish to search" webjob](https://trello.com/c/1gILFshz/505-better-logging-for-bulk-publish-to-search-webjob)

**Pull request merged**